CMSgov/mint-app
# [EASI-3688] - Claims processing precedence

# EASI-3688

## Changes and Description

[Figma](https://www.figma.com/file/M72DReE73SSXMN48WZfaKN/MINT---New-Model-Process-(Model-Intake)?type=design&node-id=7566-48414&mode=design&t=xpkoD60bsqtr1sYU-0)

BE Changes
- Added new field to schema - `claimsProcessingPrecedence`
- Added `Other`, and `Note` string fields
- Added migration to add column on `plan_payments`
- Added test for new field

FE Changes
- Added new field `claimsProcessingPrecedence` to queries and forms
- Added to Readonly and CSV
- Updated unit tests
- Added cypress test



## How to test this change

- Navigate to page 2 of payments
- Make selections on claims processing precedence question
- Verify data persists on save
- Verify render on readonly
- Verify csv export data

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
